src/Sylius/Bundle/AttributeBundle/AttributeType/DateAttributeType.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\AttributeType;
+
+use Sylius\Component\Attribute\AttributeType\AttributeTypeInterface;
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+use Symfony\Component\Validator\ConstraintViolationList;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class DateAttributeType implements AttributeTypeInterface
+{
+    const TYPE = 'date';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getStorageType()
+    {
+        return AttributeValueInterface::STORAGE_DATE;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getType()
+    {
+        return static::TYPE;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function validate(AttributeValueInterface $attributeValue, ExecutionContextInterface $context)
+    {
+        return new ConstraintViolationList();
```

Comment:
@Zales0123 If it does not need to return indeed then I'd remove these, otherwise they can stay. We will improve validation (decouple it from Symfony perhaps) in next iteration.